group_project_v2/project_api/dtos.py

```diff
@@ -0,0 +1,82 @@
+from group_project_v2.utils import make_user_caption
+
+
+# pylint:disable=too-many-instance-attributes
+class UserDetails(object):
+    def __init__(self, **kwargs):
+        self.id = kwargs.get('id', None)
+        self.email = kwargs.get('email', None)
+        self.username = kwargs.get('username', None)
+        self.uri = kwargs.get('uri', None)
+        self.first_name = kwargs.get('first_name', None)
+        self.last_name = kwargs.get('last_name', None)
+        self._full_name = kwargs.get('full_name', None)
+        self.gender = kwargs.get('gender', None)
+        self.avatar_url = kwargs.get('avatar_url', None)
+        self.city = kwargs.get('city', None)
+        self.country = kwargs.get('country', None)
+        self.is_active = kwargs.get('is_active', None)
+        self.level_of_education = kwargs.get('level_of_education', None)
+        self.organization = kwargs.get('organization', None)
```

Comment:
You can replace most of this with a list (`properties = ['id', 'email', 'username', ...]`) and `for property in properties: setattr(self, kwargs.get(property))`

You could also share that code among all the classes in this file.

Also, note that you don't need the `None` default on your call to `.get()`